docs/test_scripts/test_simple_distributed.py

```diff
@@ -0,0 +1,85 @@
+# [imports]
+import torch
+from torch.nn.parallel import DistributedDataParallel
+
+import modulus
+from modulus.datapipes.benchmarks.darcy import Darcy2D
+from modulus.distributed import DistributedManager
+from modulus.metrics.general.mse import mse
+from modulus.models.fno.fno import FNO
+from modulus.utils import StaticCaptureTraining
+
+# [imports]
+
+# [code]
+
+
+def main():
+
+    DistributedManager.initialize()
```

Comment:
A comment here explaining why this is needed would be helpful

Reply:
Added referring to the distributed docs.